cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -101,4 +101,16 @@ type NodeConfiguration struct {
 	NodeName                 string
 	TLSBootstrapToken        string
 	Token                    string
+
+	// TLSDiscoveryRootCAPubKeys specifies a set of public key pins. The root CA
+	// discovered during TLS bootstrapping must match one of these values.
```

Comment:
Technically, this is during the bootstrap discovery, not TLS.  We use this so that we can establish the kubelet trusting the master.